conftest.py

```diff
@@ -87,6 +87,17 @@ def tmp_table(sql_server):
         hook.run(f"DROP SCHEMA IF EXISTS {temporary_table.schema} CASCADE;")
 
 
+@pytest.fixture
+def output_table(request):
```

Comment:
The name of this fixture might be misleading, if part of it is specific to Postgres